sycl/test/group-algorithm/all_of.cpp

```diff
@@ -0,0 +1,81 @@
+// RUN: %clangxx -fsycl -D __SYCL_INTEL_GROUP_ALGORITHMS__ %s -o %t.out
```

Comment:
Don't we enable these by default and rather disable by request?

Reply:
I've tried to address this in e932422, by swapping each `#ifdef __SYCL_INTEL_GROUP_ALGORITHMS__` for a `#ifndef __DISABLE_SYCL_INTEL_GROUP_ALGORITHMS__`.  Is that what you meant?

Reply:
Yes. Thanks!